### DISCUSSION TITLE
Quicklink Cre8/StudioPro - Infringement of the GPL?

### DISCUSSION THREAD

**Discussion by martinS1992** [obsproject/obs-studio#10585] [General]
I'd like to bring to your attention a matter that I believe warrants our consideration. Upon examination, it appears that Quicklink Cre8 might be a fork of OBS. As stewards of the GPL license, it's our responsibility to ensure compliance from all derivative projects.

I invite fellow community members to join me in investigating this potential connection. Let's engage in a constructive dialogue to verify and address any concerns regarding adherence to open-source principles.

https://www.quicklink.tv/products/studiopro/

**Comment by Fenrirthviti** [comment]
We are aware of this, and have been investigating internally.

**Reply by crazy-logic** [reply]
Any updates on this - I have a job comming up where we might be using a studiopro and it's an open source client - they would be very upset if it were found that we were using something that falls foul of licensing agreements.

**Reply by Fenrirthviti** [reply]
It is our current understanding that all offenses have been corrected. It's difficult for us to validate, as the software itself is behind a paywall of their expensive devices, but all software that includes parts of, or is based on, OBS Studio should be offering source code.

If you find otherwise, please let us know at contact@obsproject.com.
